system/Security/Security.php

```diff
@@ -184,6 +191,7 @@ public function __construct($config)
 		$this->CSRFHeaderName = $config->CSRFHeaderName;
 		$this->CSRFCookieName = $config->CSRFCookieName;
 		$this->CSRFRegenerate = $config->CSRFRegenerate;
+		$this->CSRFSameSite   = $config->CSRFSameSite;
```

Comment:
need default fallback as well:

```suggestion
              $this->CSRFSameSite   = $config->CSRFSameSite ?? 'Lax';
```

Reply:
I am not sure I understand the reasoning behind this as no other config has this check but instead is set as default values in the classes?

Reply:
the `public $CSRFSameSite` is a new config, so need a default value in case user do composer update without comparing new config in app/

Reply:
Fair point.